File: calculator.py
import sys

if len(sys.argv) < 3:
    print("Provide two numbers")
    sys.exit(1)

num1 = float(sys.argv[1])
num2 = float(sys.argv[2])

print("Addition:", num1 + num2)
print("Subtraction:", num1 - num2)
print("Multiplication:", num1 * num2)
print("Division:", num1 / num2)

File: calculator.java
public class calculator {
    public static void main(String[] args) {

        if (args.length < 2) {
            System.out.println("Please provide two numbers.");
            return;
        }

        double num1 = Double.parseDouble(args[0]);
        double num2 = Double.parseDouble(args[1]);

        System.out.println("Addition: " + (num1 + num2));
        System.out.println("Subtraction: " + (num1 - num2));
        System.out.println("Multiplication: " + (num1 * num2));
        System.out.println("Division: " + (num1 / num2));
    }
}
